plugins/plugin-core-graphing/lib/GraphingPlugin.ts

```diff
@@ -66,7 +65,7 @@ export default class GraphingPlugin extends ListenerPlugin {
       alias: [],
       default: "cfg",
       description: "The path where the csv should be saved",
-      group: CliOptionGroups.Storage,
+      group: "Graphing Options:",
```

Comment:
Should we create an enum for this, even though we only have one instance for now?

Reply:
Added it.